python/private/pypi/extension.bzl

```diff
@@ -383,6 +387,12 @@ def _configure(config, *, platform, os_name, arch_name, config_settings, env = {
             if key not in _SUPPORTED_PEP508_KEYS:
                 fail("Unsupported key in the PEP508 environment: {}".format(key))
 
+        if not os_name:
+            fail("'os_name' is required")
```

Comment:
Were os_name and arch_name effectively required before, or is this new? Though, I think this pip.defaults API is still experimental, right?